Ghidra/Features/Base/src/main/java/ghidra/app/plugin/core/eclipse/EclipseIntegrationPlugin.java

```diff
@@ -194,7 +194,7 @@ public void offerGhidraDevInstallation(TaskMonitor monitor) {
 			}
 			if (ghidraDevFile == null) {
 				Msg.showError(this, null, errorTitle,
-					"GhidraDev Eclipse extension does not exist:\n" + ghidraDevFile);
+					"GhidraDev Eclipse extension does not exist:\n" + null);
```

Comment:
Don't add null to the string.

Reply:
Yeah well at this point of the code, it must always be null. The whole appending probably doesn't really help...